packages/paste-theme/src/themeProvider.tsx

```diff
@@ -10,11 +10,15 @@ const pasteGlobalStyles = css`
   html {
     font-size: 100%;
   }
+  body {
+    margin: ${themeGet('space.space0')};
+    font-size: ${themeGet('fontSizes.fontSize30')};
+  }
```

Comment:
So, these might seem risky, but I've tried them in my custom css plugin in console, flex and marketing campaigns and it seems totally fine to do so. 

Font size not on the body results in some elements like `code` rendering incorrectly, even when given an explicit font size. It looks for the body instead of the html element for it's base. And we still want a good out of the box experience size wise.